examples/complete/outputs.tf

```
@@ -13,32 +13,47 @@ output "new_sg_name" {
   value       = module.new_security_group.name
 }
 
-output "external_sg_id" {
-  description = "The external Security Group ID"
-  value       = module.external_security_group.id
+output "new_sg_details" {
```

```suggestion
output "created_sg_details" {
```

maybe `created` is better since we create it?

---

This is just the example, so I don't care so much, but I have no object to changing the name to `created` other than it ripples through into the test cases.